src/S_Ecobee1.xml

```diff
@@ -15,6 +15,12 @@
       <dataType>string</dataType>
       <sendEventsAttribute>yes</sendEventsAttribute>
     </stateVariable>
+    <stateVariable>
+      <name>FanMinOnTime</name>
+      <dataType>i4</dataType>
+      <defaultValue>0</defaultValue>
+      <sendEventsAttribute>yes</sendEventsAttribute>
+    </stateVariable>
```

Comment:
I originally tried it without it because that was the way SendMessage and SetClimateHold were defined. However, i found that it doesn't work in UI5. I have not tested with UI7 (too many plugins and haven't found time to deal with upgrading all of it)

Here is a screenshot showing the issue:

![ecobee-scene](https://cloud.githubusercontent.com/assets/5420432/14559772/2bec8eb8-02d9-11e6-853a-42fcd187fca0.png)

I was going to bring this up in the pull request (or as a separate issue), but forgot about it when I actually submitted it.